services/TS29222_CAPIF_Auditing_API/logs/controllers/default_controller.py

```diff
@@ -37,22 +51,30 @@ def api_invocation_logs_get(aef_id=None, api_invoker_id=None, time_range_start=N
     :param resource_name: Name of the specific resource invoked.
     :type resource_name: str
     :param src_interface: Interface description of the API invoker.
-    :type src_interface: dict | bytes
+    :type src_interface: str
     :param dest_interface: Interface description of the API invoked.
-    :type dest_interface: dict | bytes
+    :type dest_interface: str
     :param supported_features: To filter irrelevant responses related to unsupported features
     :type supported_features: str
 
     :rtype: InvocationLog
     """
+
+    cert_tmp = request.headers['X-Ssl-Client-Cert']
+    cert_raw = cert_tmp.replace('\t', '')
+
+    cert = x509.load_pem_x509_certificate(str.encode(cert_raw), default_backend())
```

Comment:
you can remove this code.
We are talking how to implement the identity verification in nginx using the DN in certificates